index.html

```diff
@@ -368,20 +388,71 @@ <h3>
           <p class="note" title="HTTP API">
 There is no HTTP API specified for the update process leaving programmatic
 registrations and management to be defined by each implementation.
-          </p>
+</p>
         </section>
 
         <section>
           <h3>
 Deactivate (Revoke)
           </h3>
           <p>
-To delete the DID document, the <code>did.json</code> has to be removed or has
-to be no longer publicly available due to any other means.
+To delete the DID document, the associated file or files have to be removed or
+made no longer publicly available due to any other means.
           </p>
         </section>
       </section>
 
+      <section>
+      <h2>
+Webserver Configuration
+      </h2>
+      <h3>
+Index Files
+      </h3>
+      <p>
+Webserver software typically supports returning specified files when a URL path
+matches a directory rather than a file. These are generally refered to as index
+files, and <code>index.html</code> is a common default. To resolve method specific
+identifiers that have no filename and extension, webservers MUST be configured
+to return the appropriate DID file when a request path matches a DID directory.
+      </p>
+      <h3>
+MIME Types
+      </h3>
+      <p>
+Webservers MUST be configured to associate proper content types with registered
+filename extensions. The filename extensions and content types are noted in the
+[[DID-CORE]] <a href="https://w3c.github.io/did-core/#iana-considerations">IANA
+considerations section</a>.
+      </p>
+      <h3>
+Multiple DID Representations
+      </h3>
+      <p>
+DIDs in directories MAY support multiple representations. The server MUST support
+at least one of the DID representatons listed in the note above, and MAY support
+more than one. If multiple representations are supported, the server MUST prefer
+the representation in the HTTP <code>Accept:</code> request header when selecting
+an index file to return.
+      </p>
+      <h3>
+Cross Origin Browser Access
+      </h3>
+      <p>
+Webservers MUST be configured with appropriate CORS headers if web browser
+clients operating in other domains will need access to the DID documents. [[CORS]]
+specifies headers to enable browser cross-domain access.
+      </p>
```

Comment:
Too vague to be implementable. Specific CORS headers should be listed here.

Reply:
I can add examples.

Reply:
To be clear, examples don't help. You are saying that Web servers ***MUST*** do something, but are then being vague about what they must do. Every MUST statement we have in the specification will be translated into a test that either returns PASS or FAIL for every implementation being tested. The problem here is that you're using words like "appropriate" -- well, that's really hard to test because you're being vague about what is expected. 

If you are going to use normative MUST language, think about how you're going to convert that into a test that all implementations will either pass or fail. We need to get hard requirements down to objective testability if the spec is going to be useful to implementers.

Reply:
The normative language is not appropriate here.

Requirements for CORS and Cache-Control headers will vary by implementation, and cannot be tested globally.

Revised.
e308e221ac122163819a15aad4774a737dd4b64f